### DISCUSSION TITLE
Resume playing after place the same card again

### DISCUSSION THREAD

**Discussion by Schmogg** [MiczFlor/RPi-Jukebox-RFID#2431] [Q&A]
Hi, 
I have an Rpi 3B+ that has a PN532 NFc HAT on it. On the Pi is the current V3 with “place not swipe” enabled. Everything works quite well so far, but I would like it to resume playing at the same point when I remove the card and put it back on. is this possible? Sorry for the bad english

**Comment by AlvinSchiller** [comment]
Hi,
did you activate the "second swipe"? I read in some tickets that in gerenal this behavior should work. But i cant test it mayself at the moment.


The resume feature for a specific card or folder is missing right now in v3 :/
There a quite a lot Issues that address this and have some implementation solution, yet none has a final state to provide right now.

See here and the linked tickets:
https://github.com/MiczFlor/RPi-Jukebox-RFID/pull/2330
https://github.com/MiczFlor/RPi-Jukebox-RFID/issues/1945

**Reply by Schmogg** [reply]
thanks for the reply. Can you tell me where I activate second swipe? And is possible to do it in v2?

**Reply by AlvinSchiller** [reply]
For v3 "second swipe" with mode "toggle" should be activate by default. To validate check your [jukebox.yaml](https://github.com/MiczFlor/RPi-Jukebox-RFID/blob/0ab75c1d28955e85c2069503b5a8428a69967d0d/resources/default-settings/jukebox.default.yaml#L82C3-L82C22). i think the webapp doesn't feature this option yet.

In v2 “place not swipe” and "second swipe" can be set in the Settings menu . The Resume Option is activatible on the Folder (like shuffle, repeat, ...).


Do you maybe have some logs, in case there is something different causing this behavior?

**Comment by moonbit5** [comment]
so is PLACENOTSWIPE available at all in V3 currently?

**Reply by AlvinSchiller** [reply]
Yes it is available, but depends on your specific reader if usable. 

https://github.com/MiczFlor/RPi-Jukebox-RFID/blob/future3/main/documentation/developers/rfid/README.md

**Reply by moonbit5** [reply]
perfect thank you, I missed that!
